src/index.js

```diff
@@ -0,0 +1,162 @@
+const state = {
+  city: 'Neema & Pavi Land',
+  lat: 47.6038321,
+  long: -122.3300624,
+  temp: 73,
+};
+
+const convertKtoF = (temp) => {
+  return (temp - 273.15) * (9 / 5) + 32;
+};
+
+const displayCityInfo = (cityinfo) => {
+  const cityInfo = document.getElementById('cityInfo');
+  cityInfo.textContent = `Current Temperature for: ${cityinfo}`;
+};
+
+const displayCityError = () => {
+  const cityInfo = document.getElementById('cityInfo');
+  cityInfo.textContent = "Couldn't find temperature for this city!";
+};
+
+const clearCityInfo = () => {
+  const cityInfo = document.getElementById('cityInfo');
+  cityInfo.textContent = '';
+};
+
+const findLatAndLong = () => {
+  //let lat, long;
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: state.city,
+      },
+    })
+    .then((response) => {
+      console.log(response.data);
+      state.lat = response.data[0].lat;
+      state.long = response.data[0].lon;
+      const cityinfo = response.data[0].display_name;
+      getWeather(cityinfo);
+    })
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+      displayCityError();
+    });
+};
+
+const getWeather = (cityinfo) => {
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: state.lat,
+        lon: state.long,
+      },
+    })
+    .then((response) => {
+      const weather = response.data;
+      state.temp = Math.round(convertKtoF(weather.main.temp));
+      formatTempAndGarden();
+      displayCityInfo(cityinfo);
+    })
+    .catch((error) => {
+      console.log('Error getting the weather:', error);
+      displayCityError();
+    });
+};
```

Comment:
These axios calls look great. In the future, think about how you could use await here to your advantage!